Monika After Story/game/chess.rpy

```diff
@@ -551,7 +656,10 @@ label game_chess:
         failed_to_load_save = True
 
         #Prelim definitions of the rules for the menu later
-        do_really_bad_chess = False
+        chessmode = "normal_chess"
+        NormalChessSelecting = True
+        Chess960Selecting = False
+        BadChessSelecting = False
```

Comment:
`chessmode` is enough of a var to differentiate between the selected chess mode.

I don't think we need the `xSelecting` vars since:
1. They don't change after being set
2. They are only used to control the appearance of the button in the menu, which is fine to directly put `True/False` in the options tuple